crates/dbsp/src/storage/backend/mod.rs

```diff
@@ -303,7 +303,9 @@ pub fn new_default_backend(tempdir: PathBuf, cache: StorageCacheConfig) -> Backe
             }
         }
     } else {
-        info!("io_uring backend disabled by default (set FELDERA_USE_IO_URING=1 to enable");
+        tracing::info!(
+            "io_uring backend disabled by default (set FELDERA_USE_IO_URING=1 to enable"
```

Comment:
Missing a closing brace:  "io_uring backend disabled by default (set FELDERA_USE_IO_URING=1 to enable)"